package.json

```diff
@@ -19,16 +19,19 @@
     "test": "lerna run test",
     "server": "nodemon packages/duckdb/bin/run-server.js",
     "dev": "vite",
-    "release": "npm run test && npm run lint && lerna publish"
+    "release": "npm run build && npm run test && npm run lint && lerna publish"
```

Comment:
Could we run tests with ts-node so we don't need to build?

Reply:
Looks like we use ts-mocha so this should not be needed, right?

Reply:
@domoritz the reason I'd added was because the `mosaic-core` package relies on this library for testing. my thinking was that running the `build` first would ensure that mosaic-core is referencing the `dist` file, but maybe that's not right?

Reply:
Oh I see. Maybe we should run the tests everywhere with ts-mocha so that we can use ts or js.

Reply:
yeah, that's a solid idea. I think it'd be easy enough to add too